rules/manager.go

```diff
@@ -232,6 +235,7 @@ func (g *Group) run(ctx context.Context) {
 
 		iterationDuration.Observe(timeSinceStart.Seconds())
 		g.SetEvaluationDuration(timeSinceStart)
+		g.SetEvaluationTimestamp(time.Now())
```

Comment:
This line should be `g.SetEvaluationTimestamp(start)`.

Reply:
Hey Wei,

That is similar to how I originally had it, but after discussing with Julius we agreed that it makes more sense and is more consistent to set the evaluation timestamp to the time that evaluation completed rather than when it started.

Reply:
There may have been a misunderstanding here: I meant we should still set it to the start / eval timestamp, but only after the evaluation has finished.

Reply:
Basically, this will tell people the timestamp for which they can expect rule result samples to be in the TSDB already, which I think is the most useful timestamp to expose for users.

Reply:
Ah, yes -- I misunderstood. I can update this today to use the `start` variable, then.